content/en/docs/Advanced/configuring_partitions.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Configuring partitions"
+linkTitle: "Configuring partitions"
+weight: 1
+description: >
+---
+
+{{% alert title="Note" color="warning" %}}
+
+This feature will be available in Kairos version `2.4.0` and in all future releases.
+
+{{% /alert %}}
+
+
+Kairos configuration mechanism for partitions is based on the [cloud-config]({{< relref "../Reference/configuration" >}}) file 
+given during installation to override the default values set by the installer.
+
+We allow certain flexibility in the sizes and filesystems used for the default install and allow to create extra partitions as well.
+
+For example, the following cloud-config will make the `oem` partition have a size of 512Mb and an `ext4` filesystem, 
+recovery with a size of 10000Mb and a `ext4` filesystem, while leaving the rest of the partitions to their default sizes and filesystems. 
+
+```yaml
+#cloud-config
+
+install:
+  device: "/dev/sda"
+  auto: true
+  partitions:
+    oem:
+      size: 512
+      fs: ext4
+    recovery:
+      size: 10000
+      fs: ext4
+```
+
+
+
+And the following config will leave the default partitions as is, but create 2 new extra partitions with the given sizes, filesystems and labels:
+
+```yaml
+#cloud-config
+
+install:
+  device: "/dev/sda"
+  auto: true
+  extra-partitions:
+    - name: first_partition
+      size: 512
+      fs: ext3
+    - name: second_partition
+      size: 100
+      fs: ext2
+      label: PARTITION_TWO
+```
+
+In the case of wanting an extra partition to take over the rest of the disk space (by setting the size to 0), we need to also mark persistent with a fixed size.
+Currently, the default value for the persistent partition is set to 0 which means "take over the rest of the disk space free".
+In order to mark a different partition with that value, we need to set it to a fixed size, as there cannot be 2 partitions that want to take over the rest of the space.
```

Comment:
I think this sounds a bit more complex than it needs to be. Maybe something like: only one partition can expand to the rest of the disk. Either persistent or one of the extra-partitions. In case you want the latter, you need to specify the size of persistent.